src/main/java/ScrabbleGame.java

```diff
@@ -0,0 +1,67 @@
+import java.util.*;
+
+public class ScrabbleGame {
+
+    private List<String> playedWords = new ArrayList<>();
+    private List<Character> tiles = new ArrayList<>();
+    private ScrabbleDictionary dictionary = new ScrabbleDictionary();
+    private Random rnd = new Random();
+
+
+    public ScrabbleGame() {
+        initializeTiles();
+    }
+
+    private void initializeTiles() {
+        for (int i = 0; i < 7; i++) {
+            tiles.add((char) (rnd.nextInt(26) + 'a'));
+        }
+    }
+
+    public void playGame(Scanner scanner) {
+        while (true) {
+            System.out.println("Enter word with given letters or 'exit' to end game\n" + tiles);
+            String word = scanner.nextLine().toLowerCase();
+            if (word.equalsIgnoreCase("exit")) {
+                System.exit(0);
+            } else if (!playWord(word)) {
+                System.out.println("invalid word. try again: ");
+            }
+        }
+    }
+
+    /**
+     * If the word exists in the ScrabbleDictionary, the letters exist in the tiles List,
+     * and playedWords does not yet contain the word,
+     * remove the letters from the list and add new random letters.
+     *
+     * @param word
+     */
+    public boolean playWord(String word) {
+        char[] wordLetters = word.toCharArray();
```

Comment:
The problem here is that since `playWord` is public, you can still call it outside of `playGame` and it won't have the `toLowerCase()`